bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -349,6 +349,10 @@ private function addApplicationLogNode(ArrayNodeDefinition $rootNode): void
                             ->info('Archive database name (optional). Tables will get archived to a different database, recommended when huge amounts of logs will be generated')
                             ->defaultValue('')
                         ->end()
+                        ->scalarNode('archive_db_table_storage_engine')
+                            ->info('Database storage engine to be used for archive tables (e.g. ARCHIVE, InnoDB, Aria, ...)')
+                            ->defaultValue('')
```

Comment:
```suggestion
                            ->defaultValue('archive')
```
would look more safer because the existing table would be still in that engine and to keep them consistent across the archives, or is not that relevant?

Reply:
or should we provide a migration to "convert" existing tables to the first of the Engines list? 
It may inadvertently conceal some incompatibility error (eg. the described limitations on PR comment) or so.